plugins/modules/zypper_repository_info.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Tobias Zeuch <tobias.zeuch@sap.com>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: zypper_repository_info
+author: "Tobias Zeuch"
+short_description: List Zypper repositories
+description:
+    - List Zypper repositories on SUSE and openSUSE.
+    - Note: for info about packages, use the packages ansible.builtin.package_facts
```

Comment:
1. You need to quote, since otherwise YAML parses this as a one-element dictionary wit key "Note" and value "for info ...".
2. Use `M(...)` to reference a module (see https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation for more info).
```suggestion
    - "Note: for info about packages, use the module M(ansible.builtin.package_facts)."
```

Reply:
Or, alternatively, move the comment from the `description:` section to the `notes:` one.